packages/react-router-permissions/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@tshio/react-router-permissions",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "module": "es/index.js",
+  "repository": "TheSoftwareHouse/react-packages",
+  "author": "Rafał Kucharski <rafal.kucharski@thesoftwarehouse.pl>",
+  "publishConfig": {
+    "access": "public"
+  },
+  "contributors": [
+    "Mateusz Klinowski <mateusz.klinowski@thesoftwarehouse.pl>",
+    "Łukasz Pietrek <lukasz.pietrek@thesoftwarehouse.pl>"
+  ],
+  "peerDependencies": {
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1"
```

Comment:
Are `*-dom` dependencies actually required? Maybe they could be just devDependencies for tests? (react-router supports react-native)